pymobiledevice3/cli/remote.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import logging
+import os
+from typing import Optional
+
+import click
+import inquirer
+from cryptography.hazmat.primitives.asymmetric import rsa
+from inquirer.themes import GreenPassion
+
+from pymobiledevice3.cli.cli_common import UDID_ENV_VAR, print_json, set_verbosity
+from pymobiledevice3.exceptions import NoDeviceSelectedError
+from pymobiledevice3.remote.core_device_tunnel_service import create_core_device_tunnel_service
+from pymobiledevice3.remote.remote_service_discovery import RSD_PORT, RemoteServiceDiscoveryService, \
+    get_remoted_device, get_remoted_devices
+
+logger = logging.getLogger(__name__)
+
+
+class RemoteCommand(click.Command):
```

Comment:
Maybe move the UDID selection code to a function in common and inherit command?

Reply:
what exactly do you expect to be in the common base class? as we discussed, in one its the `LockdownClient` that is returned, while the other is the `hostname`. i can implement this by having two private methods:
- `_get_device_list()`
- _get_device_from_device_list()`
is this what you suggest?